padevents/padevents.py

```diff
@@ -597,6 +610,20 @@ async def aed_remove(self, ctx, index: int):
             dmevents.pop(index - 1)
         await ctx.tick()
 
+    @autoeventdm.command(name="show")
+    async def aed_show(self, ctx, index: int):
+        """Show specifics of an autoeventdm"""
+        dmevents = await self.config.user(ctx.author).dmevents()
+        if not 0 < index <= len(dmevents):
+            await ctx.send("That isn't a valid index.")
+            return
+        de = (f"Lookup number: `{index}`\n"
```

Comment:
`ret` is a nice, noncommittal name for a returned value (short for return), if you want a useless variable name use that

I don't like 2-letter abbreviations of nonstandard "types" as variable names, it's too unfamiliar for the many people who read our code base (Aradia will tell u I yell at him about this a lot too)